loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/ValidationFormatterCsvMultilineWriter.java

```diff
@@ -18,23 +18,25 @@
  *
  * @author Massimo Ferraro <massimo.ferraro@techrain.it>
  */
-public class FlowsFormatterCsvMultilineWriter extends FlowsFormatterWriter {
+public class ValidationFormatterCsvMultilineWriter extends ValidationFormatterWriter {
 
     private static TableFormatterConfig TABLE_FORMATTER_CONFIG = TableFormatterConfig.load();
```

Comment:
Static initializer here read the user configuration folder (see #151)
The TableFormatterConfig should be pass as a parameter.